tests/spec/core/xref-spec.js

```diff
@@ -762,25 +765,32 @@ describe("Core — xref", () => {
           <pre class="idl">
           enum Foo { "dashed-thing", "" };
           </pre>
-          <p id="link1">{{ ServiceWorkerUpdateViaCache["imports"] }}</p>
+          <p id="link1">{{ ReferrerPolicy["no-referrer"] }}</p>
+          <p id="link2">{{ ReferrerPolicy[""] }}</p>
           <div data-dfn-for="Foo">
             <dfn>dashed-thing</dfn> <dfn>""</dfn>
           </div>
-          <p id="link2">{{ Foo["dashed-thing"] }} {{Foo[""]}}</p>
+          <p id="link3">{{ Foo["dashed-thing"] }} {{Foo[""]}}</p>
         </section>
       `;
-      const config = { xref: ["service-workers"], localBiblio };
+      const config = { xref: ["referrer-policy"], localBiblio };
       const ops = makeStandardOps(config, body);
       const doc = await makeRSDoc(ops);
 
       const link1 = doc.getElementById("link1");
-      expect(link1.textContent).toBe(`"imports"`);
-      expect(link1.querySelector("a").textContent).toBe("imports");
+      expect(link1.textContent).toBe(`"no-referrer"`);
+      expect(link1.querySelector("a").textContent).toBe("no-referrer");
       expect(link1.querySelector("a").href).toBe(
-        expectedLinks.get("ServiceWorkerUpdateViaCache.imports")
+        "https://www.w3.org/TR/referrer-policy/#dom-referrerpolicy-no-referrer"
+      );
+
+      const link2 = doc.getElementById("link2");
+      // expect(link2.textContent).toBe(`""`); // TODO
```

Comment:
It's coming out to be `"Referrer Policy"` 😕

Reply:
magic...